sql/v2/runtime/user_defined_functions_test.go

```diff
@@ -0,0 +1,209 @@
+/*
+ Copyright 2021 The CloudEvents Authors
+ SPDX-License-Identifier: Apache-2.0
+*/
+
+package runtime_test
```

Comment:
Since this is in a separate package for import purposes, do you think it would make sense to move it into a new (nested) directory like `/runtime/test` (which would also then have all the tck files in the same directory)>

Reply:
Yes, I believe creating a dedicated test directory would provide more flexibility for future customizations. Will be right back.

Reply:
Tests assets moved to `runtime/test` dir.